CMakeLists.txt

```diff
@@ -186,12 +186,17 @@ if(LINUX)
   find_package(udev REQUIRED)
   set(HAVE_UDEV ${UDEV_FOUND})
 
+  find_package(urcu REQUIRED)
+  set(HAVE_URCU ${URCU_FOUND})
+
```

Comment:
this should go outside of the `if(LINUX)` block (it sounds like urcu is available on bsd also)